src/tnc.rs

```diff
@@ -152,9 +155,8 @@ trait TncImpl: Send + Sync {
 }
 
 /// A local or remote TNC attached to a radio, which can send and receive frames.
-pub struct Tnc {
-    imp: Box<dyn TncImpl>,
-}
+#[derive(Clone)]
```

Comment:
Part of me would like to remove this and force the time example to explicitly share a single instance for transmitting using an `Arc`. What do you think?

Reply:
If we do it this way, wouldn't it cause issues if the user has open connections and drops the `Tnc`? I was under the impression we'd want it to shut down when `TncInner` is dropped(since each connection would hold an `Arc<Mutex<TncInner>>`

Reply:
Okay I guess the user will be holding each separate `Connection` struct which will implicitly hold the TNC open. So long as the TNC stops when the user has dropped _everything_, that seems fine.

Reply:
Sorry, I'm confused what you mean. Do you still want everything to be shut down when `Tnc` is dropped?

Reply:
Sorry that wasn't clear. No, I am okay with what you said - that dropping the `Tnc` will have no effect provided the user still has one or more `Connection`s going on.

All I meant is if the user subsequently drops those `Connection` structs too, then there should be no threads or anything left in the background inaccessible.